### ISSUE TITLE
USB Stenograph plugin with Mira A3 not working

### ISSUE THREAD

**Issue by acarlow** [openstenoproject/plover#982]
## Summary

 Whenever I try to enable the USB Stenograph all I get is: WARNING: Stenograph machine is not connected

## Reproducing

Enabling USB Stenograph with a Mira A3 plugged in via USB causes the above message.  Stenograph driver is installed and appears as expected in Device Manager.

## Plover Version

plover-weekly-v4.0.0.dev8-66.g685bd33

## System

Win 10, 1803

**Comment by morinted** [comment]
You have CaseCAT that works with this machine, right? And CaseCAT is closed
when trying to use Plover?

**Comment by acarlow** [comment]
its digitalCAT and I do have it closed.  I will go ahead and do a reboot to verify nothing is being held open.

**Comment by acarlow** [comment]
Yeah, same thing after a reboot with nothing started but plover.

**Comment by morinted** [comment]
It's strangest to me that it's working with other software and just not Plover.

First thing I'd check before digging much deeper is just reinstalling the USB drivers from the Stenograph website and then rebooting: http://www.stenograph.com/content/files/documents/USB_Writer_Drivers.zip

**Comment by acarlow** [comment]
Yes, I agree, it's strange. I'm outta town for a couple weeks but will
reinstall the driver's when I get back and update the ticket.

**Comment by acarlow** [comment]
Ok, I reinstalled the driver from the link you provided and the issue remains.  I have received the serial realtime cable that I purchased and that does work as expected with Plover.  What next?

**Comment by gdwaynewarner** [comment]
I'm a few years late to this party, but when I was using digitalCAT, I was told it wouldn't work with the KeySpan adapter. The Support Tech was quite surprised when I told him it worked just fine with the ProCAT Flash.

Can't hurt to try another adapter .....

**Comment by mkrnr** [comment]
Closing this ticket due to inactivity. Please comment if this is still an issue.
